src/graphx/graphx.asm

```diff
@@ -523,13 +525,15 @@ gfx_GetPixel:
 	inc	hl
 	inc	hl
 	inc	hl			; move to next argument
-	ld	de,0
 	ld	e,(hl)			; e = y coordinate
-	call	_PixelPtr
```

Comment:
I believe this routine does some checks first, here is it safe to get rid of that?